### ISSUE TITLE
Folder doesn't resize after disposing child/input

### ISSUE THREAD

**Issue by abolfazlshirazi** [cocopon/tweakpane#41]
Hi,
When I dispose some elements, they will disappear and the other items below it shift up and it is fine. But the height of the pane won't be updated so there will be an empty space at the end of the pane in a folder.

![Untitled-2](https://user-images.githubusercontent.com/45438565/80911986-341a9b80-8d3a-11ea-9ed2-006a51d17ea3.jpg)

**Comment by abolfazlshirazi** [comment]
I realized the followings solve the issue. However, it is just a temporary solution:

```
// rPane: The desired pane that you have
// ind: the index of the folder in the pane which you want to fix the height issue

var ind = 0;
var curH = rPane.controller.ucList_.items[ind].computeExpandedHeight_(); 
rPane.controller.ucList_.items[ind].view.folder_.expandedHeight_ = curH;
rPane.controller.ucList_.items[ind].folder.expandedHeight_ = curH;
rPane.refresh();
// You need to expand/collapse it once to make it work
rPane.controller.ucList_.items[ind].folder.expanded = false;
rPane.controller.ucList_.items[ind].folder.expanded = true;
```

**Comment by cocopon** [comment]
Fixed in #43. Thank you for reporting the issue and please wait for the next release!
